tests/performance/statistic/internal/helpers/helper.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2025 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/deckhouse/virtualization/api/client/kubeclient"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func ToSeconds(duration *metav1.Duration) float64 {
+	if duration == nil {
+		return 0
+	}
+	return duration.Seconds()
+}
+
+func CreateKubeConfig() kubeclient.Client {
+	// Get the KUBECONFIG environment variable
+	kubeconfigEnv := os.Getenv("KUBECONFIG")
+	if kubeconfigEnv == "" {
+
+		fmt.Println("Try to use default path $HOME/.kube/config")
+		userHomeDir, err := os.UserHomeDir()
+		if err != nil {
+			fmt.Println("Failed to get user home directory:", err)
+			os.Exit(1)
+		}
+		kubeconfigEnv = userHomeDir + "/.kube/config"
+
+		_ = os.Setenv("KUBECONFIG", kubeconfigEnv)
+		kubeconfigEnv = os.Getenv("KUBECONFIG")
+
+		if kubeconfigEnv == "" {
+			fmt.Println("KUBECONFIG environment variable is not set. Exiting.")
+			os.Exit(1)
+		}
+	}
+
+	// Split the KUBECONFIG environment variable (handles merged kubeconfig paths)
+	kubeconfigPaths := strings.Split(kubeconfigEnv, string(os.PathListSeparator))
+	if len(kubeconfigPaths) == 0 {
+		fmt.Println("No valid kubeconfig paths found in KUBECONFIG. Exiting.")
+		os.Exit(1)
+	}
+
+	fmt.Printf("Using KUBECONFIG paths: %v\n", kubeconfigPaths)
+
+	// Load the kubeconfig from the merged paths
+	loadingRules := &clientcmd.ClientConfigLoadingRules{
+		Precedence: kubeconfigPaths,
+	}
+	clientConfig, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		loadingRules,
+		&clientcmd.ConfigOverrides{},
+	).ClientConfig()
+	if err != nil {
+		fmt.Printf("Failed to load kubeconfig: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Create a Kubernetes client
+	client, err := kubeclient.GetClientFromRESTConfig(clientConfig)
+	if err != nil {
+		fmt.Printf("Failed to create Kubernetes client: %v\n", err)
+		os.Exit(1)
+	}
+	return client
+}
+
+func DurationToString(d *metav1.Duration) string {
+	if d == nil {
+		return ""
+	}
+	dur := fmt.Sprintf("%d:%d:%d", int64(d.Duration.Hours()), int64(d.Duration.Minutes()), int64(d.Duration.Seconds())%60)
+	return dur
```

Comment:
```suggestion
	return dur.Truncate(time.Second).String()
```
If time should be human friendly (3h42m20s).

Or, if time is parsed by scripts, also divide minutes:

```
	dur := fmt.Sprintf("%d:%d:%d", int64(d.Duration.Hours()), int64(d.Duration.Minutes())%60, int64(d.Duration.Seconds())%60)
	return dur
```

Reply:
dur.Truncate(time.Second).String() is not suitable because the data is then processed through Excel.
The average is calculated as is, but you can plot graphs using virtual machines.
If use dur.Truncate(time.Second).String(), than we have format like this `performance-9.3m2s,12s,2m50s` - Excel doesn't parse it without additional processing.

Reply:
Ok, add `%60` for correct minutes.